dev-requirements.txt

```diff
@@ -1,6 +1,2 @@
--e git+git://github.com/plotly/dash.git@master#egg=dash
--e git+git://github.com/plotly/dash-core-components@fix-select-css#egg=dash-core-components
```

Comment:
fix-select-css got through the review it seems

Reply:
yeah I noticed that Friday https://github.com/plotly/dash-core-components/pull/471